src/Components/Cart/Cart.jsx

```diff
@@ -0,0 +1,60 @@
+
+import React from 'react'
+import Navbar from '../Navbar/Navbar-comp';
+import "./Cart.css"
+
+function Cart() {
+  return (
+      <>
+      <Navbar />
+    <div className="cart_container">
+      <div className="cart">
+          <div className="products">
+              <div className="product_1">
+                  <img src="https://www.mangaldeep.co.in/image/cache/data/multi-color-sleeveless-ruffle-kids-dress-32163-297x408.jpg" alt=""/>
```

Comment:
Give the `alt` attribute a value or else would give error. Also giving alt attribute a value helps in accessibility.